xla/stream_executor/rocm/rocm_dnn.cc

```diff
@@ -3551,6 +3576,18 @@ bool MIOpenSupport::GetMIOpenConvolveAlgorithmsFindMode(
   ScopedConvolutionDescriptor conv{convolution_descriptor,
                                    ToMIOpenDataType(element_type)};
 
+  bool is_backprop = ((kind == dnn::ConvolutionKind::BACKWARD_DATA) || 
+                              (kind == dnn::ConvolutionKind::BACKWARD_FILTER));
+  //bool is_backprop = (call_context == dnn::CallContext::kBackpropData) ||
```

Comment:
I guess these two lines that are commented out can be removed?

Reply:
Done!